docs/output.md

```diff
@@ -306,6 +306,54 @@ print(repr(result.output))
 
 _(This example is complete, it can be run "as is")_
 
+##### Handling Multiple Output Tool Calls
```

Comment:
Can this be a `!!! note` section, above the previous example? I don't want it to show up in the ToC here